### PR TITLE
Sketch: upload to media gallery

### LINKED ISSUE #281: Sketch: add ability to upload image to media library
In the same way that, for instance, the Image block allows uploading the image when its source is an external link, we'd like to add the ability to upload the SVG image (converted previously to a bitmap) to the media gallery.

Image

#283

### PR DESCRIPTION
**Pull request by retrofox**
This PR allows uploading the sketch draw (on SVG) to the media library. For this:

* Converts the SVG data to Blob
* Upload the Blob to the media library

https://user-images.githubusercontent.com/77539/174484970-695f90af-d5c3-4ae9-925e-c0abab15e2e8.mov

Fixes #281

### CODE REVIEW

File: blocks/sketch/src/controls.js [hunk 1]
```diff
@@ -90,20 +101,54 @@ const Controls = ( {
 				<ToolbarButton
 					icon={ trash }
 					onClick={ clear }
-					label={ __( 'Clear canvas', 'sketch' ) }
+					label={ __( 'Clear canvas', 'a8c-sketch' ) }
 					disabled={ isEmpty }
 				/>
+
+				<ToolbarButton
```
Comment (mtias) [inline blocks/sketch/src/controls.js:108]: Let's try adding a separate toolbar group for the upload action.

Comment (retrofox) [reply blocks/sketch/src/controls.js:111]: addressed here https://github.com/Automattic/block-experiments/pull/283/commits/28a848344f358ed6612733f2c5bb652777fd8cce
image

**Comment by mtias** [comment]
What's left here to merge this?

**Comment by retrofox** [comment]
> What's left here to merge this?

I have [only one minor concern](https://github.com/Automattic/block-experiments/pull/283/files#diff-d661f888c5d3b37e8c45df129252be1302ddce77fc636676246d330b31dcbc1aR132):
```
url: `/wp-admin/upload.php?item=${ image.id }`, // @ToDo - Get the rute properly
```
I'm not sure whether it's a safe way to point to the media item.

Anyway, let's address it in a follow-up PR.

**Pull request merged**